test/base64.cpp

```diff
@@ -0,0 +1,36 @@
+#define BOOST_TEST_MODULE base64
+#include <boost/test/included/unit_test.hpp>
+#include "wasm_boost_test.hpp"
+
+#include <xxhr/util.hpp>
+
+std::map<std::string, std::string> test_data {
+  { "", "" },
+  { "TQ=="         , "M" },
+  { "TWE="         , "Ma" },
+  { "TWFu"         , "Man" },
+  { "cGxlYXN1cmUu" , "pleasure." },
+  { "bGVhc3VyZS4=" , "leasure." },
+  { "ZWFzdXJlLg==" , "easure." },
+  { "YXN1cmUu"     , "asure." },
+  { "c3VyZS4="     , "sure." },
+  { "c3VyZS4="     , "sure." },
```

Comment:
Same data twice

Reply:
Oops I kept it. It's incidental, but it ensure there is no statefulness of the decode/encode functions. Let's keep it.